delete_expired_objects/delete_expired_objects_job.py

```diff
@@ -28,19 +35,24 @@
         limit=LIMIT_SIZE,
         offset=offset_position,
     )
-    guid_list += list(response_dict.values())
+    guid_list += [
+        record["guid"]
+        for record in response_dict.values()
+        if record["date_to_delete"] < time()
+    ]
 exception_counter, delete_counter = 0, 0
 object_api = object.Gen3Object(auth)
 
-for obj in list(response_dict.values()):
-    if obj["date_to_delete"] < time():
-        try:
-            print(f"Deleting object with guid -- {obj['guid']}")
-            object_api.delete_object(guid=obj["guid"], delete_file_locations=True)
-            delete_counter += 1
-        except Exception as ex:
-            exception_counter += 1
-            print(f"Couldn't delete object with guid -- {obj['guid']}\n{ex}")
+for obj in guid_list:
+    try:
+        print(f"Deleting object with guid -- {obj['guid']}")
+        object_api.delete_object(guid=obj["guid"], delete_file_locations=True)
+        delete_counter += 1
+    except Exception as ex:
+        exception_counter += 1
+        print(
+            f"Couldn't delete object with guid -- {obj['guid']}\n{traceback.print_exc()}"
```

Comment:
you should be calling `traceback.print_exc()` outside of the print statement, i'm not sure what printing the output of `traceback.print_exc()` does - it's probably `None`?